nrf_modem/doc/gnss_interface.rst

```diff
@@ -84,15 +84,15 @@ The data to be deleted is selected using a bitmap.
                  NRF_MODEM_GNSS_DELETE_LAST_GOOD_FIX |
                  NRF_MODEM_GNSS_DELETE_GPS_TOW |
                  NRF_MODEM_GNSS_DELETE_GPS_WEEK |
-                 NRF_MODEM_GNSS_DELETE_UTC_DATA;
+                 NRF_MODEM_GNSS_DELETE_UTC_DATA |
+                 NRF_MODEM_GNSS_DELETE_GPS_TOW_PRECISION;
 
    err = nrf_modem_gnss_nv_data_delete(delete_mask);
 
 .. note::
 
-   TCXO offset data is not obtained from satellites or A-GPS assistance.
-   It is an internal value for the GNSS.
-   It should not be deleted when simulating a cold start.
+   TCXO offset is a characteristic of each device that changes slowly over time.
```

Comment:
```suggestion
   TCXO offset is a characteristic of each device and it changes slowly over time.
```
Without this change, the sentence would suggest that each device changes slowly over time.

Reply:
How about:
```suggestion
   TCXO offset is a slowly changing characteristic of each device.
```